tsconfig.json

```diff
@@ -1,13 +1,25 @@
 {
-    "compilerOptions": {
-        "target": "es6",
-        "module": "commonjs",
-        "moduleResolution": "node",
-        "outDir": "./dist",
-        "rootDir": "./src",
-        "strict": true,
-        "noImplicitAny": true,
-        "esModuleInterop": true
-    },
-    "exclude": ["node_modules"]
+	"compilerOptions": {
+		"allowJs": false,
+		"allowUnreachableCode": false,
+		"allowUnusedLabels": false,
+		"esModuleInterop": true,
+		"exactOptionalPropertyTypes": true,
+		"forceConsistentCasingInFileNames": true,
+		"importsNotUsedAsValues": "error",
+		"isolatedModules": true,
+		"module": "esnext",
+		"moduleResolution": "node",
+		"noEmit": true,
+		"noEmitOnError": true,
+		"noFallthroughCasesInSwitch": true,
+		"noImplicitReturns": true,
+		"noUncheckedIndexedAccess": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true,
+		"outDir": "dist/",
+		"rootDir": "src/",
```

Comment:
Comparing to the previous version of this, we had the `./` prefix to suggest to look for the current directory.

I'm asking this as a sanity check, but does removing this have any side effects? Are there any chances that with this config, the project to look at the wrong place?

Reply:
Good question; in this case both with the `./` prefix and without it will work in the exact same way. I did use the `./` prefix consistently in the `package.json` when referencing folders and files, so it is probably good practice to do the same here in `tsconfig.json` as well.

Reply:
Resolved via https://github.com/1Password/load-secrets-action/pull/34/commits/6fa1be2536550171da85d05f890fad7394182d76 commit.